socs/agents/acu/agent.py

```diff
@@ -1765,6 +1850,11 @@ def add_agent_args(parser_in=None):
     pgroup.add_argument("--exercise-plan")
     pgroup.add_argument("--no-processes", action='store_true',
                         default=False)
+    pgroup.add_argument("--ignore-axes",
+                        help="Comma-delimited list of axes to ignore "
+                        "(el, az, third).")
```

Comment:
This can be limited to only accept "el", "az", and "third" by adding `choices=["el", "az", "third"], nargs="+"`. Passing args then looks more like this though:
```
--ignore-axes "el" "az" "third"
```

Reply:
Ya, ok ... I am suspicious of this syntax but I guess I can't find a logical problem with it.